setup.cfg

```diff
@@ -3,5 +3,5 @@ tag_build = .dev
 
 [flake8]
 max-line-length = 88
-ignore = E501,W503
+ignore = E203,E501,E741,W503
```

Comment:
I can see why [E741](https://www.flake8rules.com/rules/E741.html) has been added, but why add [E203](https://www.flake8rules.com/rules/E203.html)?

Reply:
black was generating code that conflicted with it, and I'd rather disable this and keep black than the other way round